kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/sessiontracker/backend/BackendConnector.java

```diff
@@ -9,14 +9,24 @@
  */
 package com.vaadin.kubernetes.starter.sessiontracker.backend;
 
+import java.time.Duration;
+
 public interface BackendConnector {
     void sendSession(SessionInfo sessionInfo);
 
     SessionInfo getSession(String clusterKey);
 
     void deleteSession(String clusterKey);
 
+    /**
+     * @deprecated use {@link #markSerializationStarted(String, Duration)}
+     */
+    @Deprecated(since = "2.4", forRemoval = true)
     void markSerializationStarted(String clusterKey);
 
+    void markSerializationStarted(String clusterKey, Duration timeToLive);
```

Comment:
To ensure backward compatibility, we can implement this as a default method, invoking the deprecated method internally.  A corresponding warning message in the log will prompt developers to migrate their code (e.g. `markSerializationStarted(String clusterKey) is deprecated and will be removed. Please implement the proper markSerializationStarted(String clusterKey, Duration timeToLive) method in ${getClass().getName()}`.

Reply:
We've discussed that we won't add any deprecated method in this case. I'm closing this comment.